src/ResourceManager/Compute/Commands.Compute.Test/Common/ComputeTestController.cs

```diff
@@ -96,6 +97,10 @@ public void RunPsTest(params string[] scripts)
             string callingClassType,
             string mockName)
         {
+            Dictionary<string, string> d = new Dictionary<string, string>();
```

Comment:
could u move this code to `RMTestBase` instead of duplicating it everywhere in the code?